libs/blocks/modal/modal.js

```diff
@@ -65,6 +66,10 @@ export function closeModal(modal) {
   const hashId = window.location.hash.replace('#', '');
   if (hashId === modal.id) window.history.pushState('', document.title, `${window.location.pathname}${window.location.search}`);
   isDelayedModal = false;
+  if (prevHash) {
```

Comment:
How does this work? If _prevHash_ is defined as empty string in L15, wouldn't this _if_ always pass as `true`?

Reply:
We set the `prevHash` value on line 263 in the event listener which gets executed on the hash change. So when the `closeModal` function gets called, the `prevHash` may have a value